src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -111,6 +114,42 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
                 .orElse(personToEdit.getMatriculationNumber());
         CovidStatus updatedCovidStatus = editPersonDescriptor.getCovidStatus()
                 .orElse(personToEdit.getStatus());
+
+        if (editPersonDescriptor.getName().isPresent()) {
+            if (updatedName.equals(personToEdit.getName())) {
+                throw new CommandException(MESSAGE_SAME_STATUS);
+            }
+        }
+
+        if (editPersonDescriptor.getPhone().isPresent()) {
+            if (updatedPhone.equals(personToEdit.getPhone())) {
```

Comment:
Try to avoid using multiple if statements to ensure SLAPs. If possible, you can try to abstract the inner or outer if statement to a predicate. 👍

Reply:
Thank you for pointing out the mistake! SLAPs is very important in the software engineering project and I have followed them.